guides/source/asset_pipeline.md

```diff
@@ -868,218 +904,348 @@ each asset will have a different file name when it is changed. This way you
+   development.
+4. In production, the gem hooks into the `assets:precompile` task. It processes
+   your Tailwind CSS files and generates optimized, production-ready
+   stylesheets, which are then included in the asset pipeline. The output is
+   fingerprinted and cached for efficient delivery.
+
+#### When Should You Use It?
+
+`tailwindcss-rails` is ideal for Rails applications that:
+
+- Want to use [Tailwind CSS](https://tailwindcss.com/) without introducing a
+  Node.js dependency or JavaScript build tools.
+- Require a minimal setup for managing utility-first CSS frameworks.
+- Need to take advantage of Tailwind's powerful features like custom themes,
+  variants, and plugins without complex configuration.
+
+The gem works seamlessly with Rails' asset pipeline tools, like Propshaft,
+ensuring that your CSS is preprocessed, digested, and efficiently served in
+production environments.
+
+### `importmap-rails`
+
+[`importmap-rails`](https://github.com/rails/importmap-rails) enables a
+Node.js-free approach to managing JavaScript in Rails applications. It leverages
+modern browser support for [ES
+Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
+to load JavaScript directly in the browser without requiring bundling or
+transpilation. This approach aligns with Rails' commitment to simplicity and
+convention over configuration.
+
+#### How `importmap-rails` Works
+
+- After installation, `importmap-rails` configures your Rails app to use
+  `<script type="module">` tags to load JavaScript modules directly in the
+  browser.
+- JavaScript dependencies are managed using the `bin/importmap` command, which
+  pins modules to URLs, typically hosted on CDNs like
+  [jsDelivr](https://www.jsdelivr.com/) that host pre-bundled, browser-ready
+  versions of libraries. This eliminates the need for `node_modules` or a
+  package manager.
+- During development, there’s no bundling step, so updates to your JavaScript
+  are instantly available, streamlining the workflow.
+- In production, the gem integrates with Propshaft to serve JavaScript files as
+  part of the asset pipeline. Propshaft ensures files are digested, cached, and
+  production-ready. Dependencies are versioned, fingerprinted, and efficiently
+  delivered without manual intervention.
+
+**NOTE**: While Propshaft ensures proper asset handling, it does not handle
+JavaScript processing or transformations — `importmap-rails` assumes your
+JavaScript is already in a browser-compatible format. This is why it works best
+for projects that don't require transpiling or bundling.
+
+By eliminating the need for a build step and Node.js, `importmap-rails`
+simplifies JavaScript management while adhering to Rails conventions.
+
+#### When Should You Use It?
+
+`importmap-rails` is ideal for Rails applications that:
+
+- Do not require complex JavaScript features like transpiling or bundling.
+- Use modern JavaScript without relying on tools like [Babel
```

Comment:
Should this `Babel` have a link to something? Looks like it might have been chopped off.

Reply:
It did 🤔 Wonder how that happened! WIll fix!